packages/field/src/FormatMixin.js

```diff
@@ -226,6 +237,49 @@ export const FormatMixin = dedupeMixin(
         this._calculateValues({ source: 'serialized' });
       }
 
+      _onValueChanged({ value }) {
+        if (super._onValueChanged) super._onValueChanged();
```

Comment:
Use brackets { 

}

Note: should probably set a linting rule to make it consistent, in Poland it was pointed out by someone that we are inconsistent on this.

Reply:
fully agree with having a linter rule active for this

I saw too many times how this ended up with bugs eventually

Reply:
I think we'll work on this later, there is a lot to change here and this MR is not about removing all the technical debt, but rather fixing one specific issue with the logic, so most of the code is just moved from one file to another

Reply:
Agreed. I would agree with a linting rule for this